retriever/lib/engine_tools.py

```diff
@@ -162,6 +164,25 @@ def xml2csv(input_file, outputfile=None, header_values=None, row_tag="row"):
     return outputfile
 
 
+def xlsxcsv(book):
+    workbook = xlrd.open_workbook(book)
+    dire = book.rstrip(".xlsx")
+    if not os.path.exists(dire):
+        os.makedirs(dire)
+    else:
+        pass
+    os.chdir(dire)
+    res = len(workbook.sheet_names())
+    for sheet in range(0, res):
```

Comment:
I think we can omit the `0` and just use `range(res)`.

Reply:
:+1: